file_store/src/file_store.rs

```diff
@@ -6,16 +6,44 @@ use aws_sdk_s3::{types::ByteStream, Client, Endpoint, Region};
 use chrono::{DateTime, Utc};
 use futures::{stream, StreamExt, TryFutureExt, TryStreamExt};
 use http::Uri;
+use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 use std::{env, path::Path};
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Settings {
+    /// Bucket name for the store. Required
+    pub bucket: String,
```

Comment:
How does the file store configure multiple buckets for services that have both an input and an output or even multiples of each? Like the iot verifier will have the input bucket for ingest reports, and output bucket for valid/invalid ingest reports, an output bucket for verified iot reward shares, and an output bucket indirectly managed for the density scaler’s periodic hex scaling factor global map?